include/SubPanel/tpls/SubPanelCheckbox.tpl

```diff
@@ -0,0 +1,53 @@
+{*
+/**
+ *
+ * SugarCRM Community Edition is a customer relationship management program developed by
+ * SugarCRM, Inc. Copyright (C) 2004-2013 SugarCRM Inc.
+ *
+ * SuiteCRM is an extension to SugarCRM Community Edition developed by SalesAgility Ltd.
+ * Copyright (C) 2011 - 2017 SalesAgility Ltd.
```

Comment:
Minor thing but this copyright should be:

```suggestion
 * Copyright (C) 2011 - 2020 SalesAgility Ltd.
```

Reply:
Thanks!